labgrid/driver/sshdriver.py

```diff
@@ -8,18 +8,88 @@
 import subprocess
 import tempfile
 import time
+import pexpect
+import shlex
```

Comment:
`shlex` is an import from the Python Standard Library, so move it up (standard imports first, then third party, then local package)